### ISSUE TITLE
Type checker conflict with Config in ModelSchema (and Schema)

### ISSUE THREAD

**Issue by paolodina** [eadwinCode/ninja-schema#29]
Hi eadwinCode!

I've encountered a minor issue when using `ModelSchema` in the `Config` inner class. Pylance reports a conflict because `ModelSchema` already defines `Config`. Here's a minimal example that triggers the error:

```python
class UserOutSchema(ModelSchema):
    class Config:
        model = User
        include = ["id", "email", "first_name", "last_name", "date_joined", "is_active"]
```

And the error message from Pylance:

```
"Config" overrides symbol of same name in class "ModelSchema"
  "project.api_extra.UserOutSchema.Config" is not assignable to "ninja_schema.orm.model_schema.ModelSchema.Config"
  Type "type[project.api_extra.UserOutSchema.Config]" is not assignable to type "type[ninja_schema.orm.model_schema.ModelSchema.Config]"
```

Would it be possible to adjust `ModelSchema` so that defining a `Config` class doesn't trigger the type checking error? The same happens with `Schema`.

Alternatively, is there a recommended way to achieve the same configuration without overriding `Config`?
